src/completers/completeState/index.js

```diff
@@ -0,0 +1,22 @@
+import isStringArray from '../validate';
+
+// Given a defaultState, it populates that state with ${key}Loading and ${key}Error
+function completeState(defaultState, ignoredTargets = []) {
+  if (!defaultState || defaultState.constructor !== Object) {
```

Comment:
use typeof here

Reply:
You can declare a `typeUtils` and declare `isObject, isString, isStringArray` there